src/utils/constants.js

```diff
@@ -0,0 +1,4 @@
+
+const BASE_URL = "https://jsonplaceholder.typicode.com/todos"
+
+export {BASE_URL};
```

Comment:
No need to export in the end, Prepending `export` before any declaration will `export` it from the file.